### DISCUSSION TITLE
creating the floorplan with LiDAR (iPad)

### DISCUSSION THREAD

**Discussion by Alino** [ExperienceLovelace/ha-floorplan#226] [Ideas]
Has anyone tried this? Is it possible? I don't have an iPad with LiDAR sensors to try this. But I know that there are some apps that can use the LiDAR to create a floor plan. If those apps can export to SVG, would it be possible to use that in ha-floorplan? Also that iPad could be then used as wall mounted dashboard with the interactive floorplan. That's my idea.

Even iPhones have LiDAR. Check out this video about Apple RoomPlan https://www.youtube.com/watch?v=wZZ7CGblV5w&t=119s
it looks to me like something that can be used to improve or create nice floorplans for ha-floorplan.

**Comment by OzGav** [comment]
It doesn’t matter how the floorplan is created. If you have it in SVG format you should be able to open it in Inkscape and then appropriately label the objects.
